docs/getting-started/using-testnet.md

```diff
@@ -0,0 +1,46 @@
+# Using Testnet Guide
+
+last updated for v1.0.0/testnet2
```

Comment:
I like the mention of testnet2 but I don't think you need the v1.0.0 version number here.  That just happens to be when testnet2 comes out but it will still be around for a fair number of versions after that.